src/Common/nunit/CommandLineOptions.cs

```diff
@@ -66,9 +75,9 @@ public CommandLineOptions(params string[] args)
                 Parse(args);
         }
         
-        #endregion
+#endregion
         
-        #region Properties
+#region Properties
```

Comment:
Let's not worry about it for now. This is the first of a series of changes that will hit this file anyway. I'll try to clean them all up in #1621